lighthouse-core/test/gather/computed/critical-request-chains-test.js

```diff
@@ -7,9 +7,9 @@
 
 /* eslint-env mocha */
 
-const GathererClass = require('../../../gather/computed/critical-request-chains');
 const assert = require('assert');
-const Gatherer = new GathererClass();
+
+const Runner = require('../../../runner.js');
```

Comment:
these tests are already cheating by assuming the exact properties of network requests the gatherer will need, so that's probably fine.